gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -265,7 +305,8 @@ def filter_by_contamination(
     data_export.export(f"{output_dir}/sample_contamination.tsv")
 
     # Filter out sample with any contamination level above 2% heteroplasmy
```

Comment:
Move comment down
```suggestion
```

Reply:
Maybe change the logger `logger.info("Removed %d samples with contamination above 2 percent", n_contaminated)` above since they are not really being removed at that point. Maybe Found instead of removed? And then, could add a logger here instead of the comment saying that they are being removed?